src/components/playback/playbackmanager.js

```diff
@@ -2267,7 +2267,7 @@ class PlaybackManager {
             if (item.IsPlaceHolder) {
                 loading.hide();
                 showPlaybackInfoErrorMessage(self, 'PlaybackErrorPlaceHolder');
-                return Promise.reject();
+                return Promise.resolve();
```

Comment:
Rejected `play`?

Reply:
It doesn't seem like we handle rejections from this function... the normal branch never rejects due to the `finally`. This fixes an unhandled promise rejection error.

Reply:
:thinking: ok so not a single place this is called handles promise rejections... I guess that is fine since it's the current state of things?